tesseract_urdf/test/tesseract_urdf_convex_mesh_unit.cpp

```diff
@@ -25,10 +25,14 @@ TEST(TesseractURDFUnit, parse_convex_mesh)  // NOLINT
   tesseract_common::GeneralResourceLocator resource_locator;
   {
     std::string str =
-        R"(<convex_mesh filename="package://tesseract_support/meshes/box_2m.ply" scale="1 2 1" extra="0 0 0"/>)";
+        R"(<tesseract:convex_mesh filename="package://tesseract_support/meshes/box_2m.ply" scale="1 2 1" extra="0 0 0"/>)";
```

Comment:
I thought convex_mesh was removed as part of this PR or is that not the case?

Reply:
Looks like I had removed this unit test from the CMakeLists but forgotten to delete the file. It's deleted now, and there should not be any more references to `<tesseract:convex_mesh>` elements